lib/events/auditlog.go

```diff
@@ -35,7 +35,6 @@ import (
 	"github.com/gravitational/teleport/lib/defaults"
 	"github.com/gravitational/teleport/lib/session"
 	"github.com/gravitational/teleport/lib/utils"
-
 	"github.com/gravitational/trace"
```

Comment:
Nit: we generally leave a blank line between the `gravitational/teleport` imports and imports from other repos to help visually identify external dependencies at a glance.